phpstan-baseline.neon

```diff
@@ -6506,6 +6506,11 @@ parameters:
 			count: 1
 			path: modules/social_features/social_event/src/Entity/EventEnrollment.php
 
+		-
+			message: "#^Method Drupal\\\\social_event\\\\Entity\\\\EventEnrollment\\:\\:getAccountEntity\\(\\) should return Drupal\\\\user\\\\UserInterface\\|null but returns Drupal\\\\Core\\\\Entity\\\\EntityInterface\\.$#"
+			count: 1
+			path: modules/social_features/social_event/src/Entity/EventEnrollment.php
```

Comment:
This can usually be solved either by fixing a mapping configuration for PHPStan Drupal or by using `assert` (the former has preference, but might not always solve the problem depending on the code).

Reply:
added [this one](https://github.com/goalgorilla/open_social/pull/3905/commits/6d069e2b2f22928f217725e1efd2a27eb5d27058#diff-0361f0c81f363476ddc6f44ab36fcbe66ee685d5f4c2a46b054924591544b766R62) to `phpstan.neon`

Reply:
reverted